src/pages/our-team.js

```diff
@@ -1,16 +1,103 @@
 import React from "react"
-import { Link } from "gatsby"
-
+import { graphql } from "gatsby"
+import get from "lodash/get"
 import Layout from "../components/layout"
 import SEO from "../components/seo"
+import TeamMember from "../components/teampage/team-member"
+import Team from "../components/teampage/teams"
+import Advisor from "../components/teampage/advisor"
+
+const TeamPage = props => {
+  const teamPage = get(props, "data.contentfulTeamPage")
+
+  console.log(teamPage)
+
+  const listOfTeamsTypes = getTeamTypes(teamPage.teamMembers)
+  console.log(listOfTeamsTypes)
+
+  /**
+   * creates a list/array of all the team types
+   * @param {Object[]} members - Array of members
+   * @param {string} members[].type - the team the member is on
+   * @returns {string[]} - Array/list of string member types ie ['board', 'marketing']
+   */
+  function getTeamTypes(members) {
+    let listOfTeamsTypes = new Set()
+
+    members.forEach(member => {
+      if (!member.team) return
+      listOfTeamsTypes.add(member.team.toLowerCase())
+    })
 
-const TeamPage = () => (
-  <Layout>
-    <SEO title="Team Page" />
-    <h1>Our Team</h1>
-    <p>Welcome to page 2</p>
-    <Link to="/">Go back to the homepage</Link>
-  </Layout>
-)
+    listOfTeamsTypes = Array.from(listOfTeamsTypes)
+    return listOfTeamsTypes
+  }
+
+  return (
+    <Layout>
+      <SEO title="Team Page" />
+      {/* <h1>{teamPage.title}</h1> */}
+
+      {listOfTeamsTypes.map((teamType, index) => {
+        const filteredByTeam = teamPage.teamMembers.filter(member => {
+          return member.team.toLowerCase() === teamType
+        })
+
+        return (
+          <Team
+            teamName={teamType}
+            members={filteredByTeam}
+            index={index}
+          ></Team>
```

Comment:
A good idea may be to introduce a .prettierrc file to the project so this stuff gets removed for you
You can look at the alleyway if you'd like an example